index.html

```diff
@@ -2,274 +2,295 @@
 
-        <section class="footer-section-end flex">
+    <!-- start about me section -->
+    <section class="about-section" id="about">
+      <div class="about-me-section">
+        <h2>About me</h2>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <button>Get My Resume</button>
 
-            <ul class="flex">
-                <li><a href="">
-                        <a href="#"><img src="images/github.png" alt="git-icon" /></a>
-                    </a></li>
-                <li><a href="">
+        <div class="border-about-me-section"></div>
+      </div>
 
-                        <a href="#"><img src="images/linkedin.png" alt="linkden-icon" /></a>
-                    </a></li>
-                <li><a href="">
+      <div class="programming-languages apply-flex">
+        <div class="language">
+          <h4 class="languages">Language</h4>
+          <ul>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+            <li></li>
+          </ul>
+        </div>
+        <div class="frameworks">
+          <h4 class="frames">Framworks</h4>
+          <ul>
+            <li>Bootstrap</li>
+            <li>RubyonRails</li>
+            <li>RSPec</li>
+            <li>CopyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+        <div class="skills">
+          <h4 class="my-skills">Skills</h4>
+          <ul>
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </div>
+    </section>
 
-                        <a href="#"><img src="images/angellist.png" alt="angelist-icon" /></a>
-                    </a></li>
-                <li><a href="">
+    <section class="contact-section apply-flex" id="contact">
+      <h3>
+        I'm always interested in hearing about new projects, so if you'd like
+        to chat please get in touch.
+      </h3>
+      <form action="https://formspree.io/f/mayvdjkb" method="POST" class="form apply-flex" id="contact-form">
+        <input type="text" name="user_fullname" placeholder="Full name" maxlength="30" />
+        <div class="full-name apply-flex">
+          <input type="text" name="first_name" placeholder="First Name" maxlength="30" />
+          <input type="text" name="last_name" placeholder="Last Name" maxlength="30" />
+        </div>
+        <input type="email" name="user_email" placeholder="Email Adress" id="contact-me-email" required />
+        <textarea name="user_message" required maxlength="500">
+Enter text here</textarea>
+        <p id="email-error" class="error-message">
+          Your email should be written in lower case.
```

Comment:
- [ ] the error message is showing in your desktop and mobile version without the user trying to submit the form. please fix that